src/model_server/lib/image_util.hpp

```diff
@@ -8,8 +8,22 @@
 
 #include <core/data/flexible_type/flexible_type.hpp>
 #include <core/storage/sframe_interface/unity_sframe.hpp>
+
 namespace turi{
 
+void copy_image_to_memory(const image_type& input, unsigned char* outptr,
+                          const std::vector<size_t>& outstrides,
+                          const std::vector<size_t>& outshape,
+                          bool channel_last);
+
+void copy_image_to_memory(const image_type& input, float* outptr,
+                          const std::vector<size_t>& outstrides,
+                          const std::vector<size_t>& outshape,
+                          bool channel_last);
+
+void image_load_to_numpy(const image_type& img, float* output,
+                         const std::vector<size_t>& outstrides);
+
 namespace image_util{
```

Comment:
PR looks good overall but could we move these functions into the `image_util` namespace?

Reply:
Done!